src/CashierFake.php

```diff
@@ -0,0 +1,271 @@
+    }
+
+    /**
+     * Format the given path into a full url.
+     *
+     * @param  string  $path
+     * @return string
+     */
+    public static function retrieveEndpoint(string $path): string
+    {
+        return Cashier::vendorsUrl()
+               .'/api/'.static::API_VERSION
+               .Str::start($path, '/');
+    }
+
+    /**
+     * Mock a given endpoint with the provided response data.
+     *
+     * @param  string  $endpoint
+     * @param  mixed  $data
+     * @return void
+     */
+    protected function mockEndpoint(string $endpoint, $data = [])
+    {
+        $response = is_array($data) ? Http::response($data) : $data;
+
+        Http::fake([
+            static::retrieveEndpoint($endpoint) => $data,
+        ]);
+    }
+
+    /**
+     * Returns the default endpoints with their fake data.
+     *
+     * @return array
+     */
+    protected function endpoints()
+    {
+        return [
+            static::PATH_PAYMENT_REFUND => [
+                'success' => true,
+                'response' => [
+                    'refund_request_id' => 12345,
+                ],
+            ],
+
+            static::PATH_SUBSCRIPTION_USERS => function () {
+                return [
+                    'success' => true,
+                    'response' => [
+                        [
+                            'subscription_id' => 3423423,
+                            'user_email' => 'john@example.com',
+                            'payment_information' => static::$paymentProvider === 'paypal'
+                                ? ['payment_method' => 'paypal']
+                                : array_merge(['payment_method' => 'card'], static::$cardData),
+                            'last_payment' => [
+                                'amount' => 0.00,
+                                'currency' => 'EUR',
+                                'date' => '',
+                            ],
+                        ],
+                    ],
+                ];
+            },
+
+            static::PATH_SUBSCRIPTION_MODIFIERS => [
+                'success' => true,
+                'response' => [
+                    [
+                        'modifier_id' => 6789,
+                        'sucscription_id' => 3423423,
+                        'amount' => 15.00,
+                        'currency' => 'EUR',
+                        'is_recurring' => false,
+                        'description' => 'This is a test modifier',
+                    ],
+                ],
+            ],
+
+            static::PATH_SUBSCRIPTION_MODIFIERS_CREATE => [
+                'success' => true,
+                'response' => [
+                    'subscription_id' => 3423423,
+                    'modifier_id' => 6789,
+                ],
+            ],
+
+            static::PATH_SUBSCRIPTION_MODIFIERS_DELETE => [
+                'success' => true,
+            ],
+        ];
+    }
+
+    const API_VERSION = '2.0';
```

Comment:
I don't see Laravel leveraging consts very often...so not sure if you'd prefer this as something else or if there's a better naming convention you'd like.

Reply:
Removed these magic strings in favour of hardcoded. Internally they're called in one place and the idea is that a user could override them without having to know the path but then that raises questions of "do we include some endpoints and not others" and then the feel of it was janky too...doing `Cashier::faker(CashierFake::PATH_SUBSCRIPTION_REFUND)` with a mix of Cashier/CashierFake looked like hot trash.